datahub-web-react/src/app/entity/user/UserGroups.tsx

```diff
@@ -1,80 +1,206 @@
+    console.log('group', userGroups, urn, pageSize);
 
     return (
-        <GroupsView direction="vertical" size="middle">
-            <Typography.Title level={3}>Group Membership</Typography.Title>
-            <Row justify="center">
-                <GroupList
-                    dataSource={userGroups}
-                    split={false}
-                    renderItem={(item, _) => (
-                        <List.Item>
-                            {entityRegistry.renderPreview(EntityType.CorpGroup, PreviewType.PREVIEW, item)}
-                        </List.Item>
-                    )}
-                    bordered
-                />
-                <Pagination
-                    current={page}
-                    pageSize={pageSize}
-                    total={total}
-                    showLessItems
-                    onChange={onChangeGroupsPage}
-                    showSizeChanger={false}
-                />
-            </Row>
-        </GroupsView>
+        <>
+            <GroupsViewWrapper>
+                <Row justify="space-between">
+                    {userGroups &&
+                        userGroups.map((item) => {
+                            return (
+                                <GroupItemColumn xl={8} lg={8} md={8} sm={12} xs={24}>
+                                    <Link to={entityRegistry.getEntityUrl(EntityType.CorpGroup, item.urn)}>
+                                        <GroupItem>
+                                            <Row style={{ padding: '9px 11px 9px 11px' }}>
+                                                <GroupTitle>{item.name}</GroupTitle>
+                                                <GroupMember>10 members</GroupMember>
+                                            </Row>
+                                            <Row style={{ padding: '2px 13px' }}>
+                                                <GroupDescription>
+                                                    {item.info?.description && item.info?.description.length > 85 ? (
+                                                        <Tooltip title={item.info?.description}>
+                                                            {`${item.info?.description.slice(0, 80)} ...`}
+                                                        </Tooltip>
+                                                    ) : (
+                                                        item.info?.description
+                                                    )}
+                                                </GroupDescription>
+                                            </Row>
+                                        </GroupItem>
+                                    </Link>
+                                </GroupItemColumn>
+                            );
+                        })}
+                    {/* dummy */}
+                    <GroupItemColumn xl={8} lg={8} md={8} sm={12} xs={24}>
```

Comment:
This component is quite long - can we further break this out into small components?

Reply:
I think its only long because there's sample data in here that is copy + pasted :)